### PR TITLE
Add unesco hackathon option for fossasia in ticket pre payment form

### PR DESCRIPTION
**Pull request by SaptakS**
@mariobehling @hpdang as discussed, I have added the option for unesco hackathon.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: migrations/versions/e9d61dbd0cb8_.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    op.create_foreign_key(None, 'orders', 'user', ['user_id'], ['id'], ondelete='SET NULL')
+    op.create_foreign_key(None, 'orders', 'discount_codes', ['discount_code_id'], ['id'], ondelete='SET NULL')
+    op.create_foreign_key(None, 'orders', 'events', ['event_id'], ['id'], ondelete='SET NULL')
+    op.create_foreign_key(None, 'session', 'tracks', ['track_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'session', 'session_type', ['session_type_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'session', 'microlocation', ['microlocation_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'session', 'events', ['event_id'], ['id'], ondelete='CASCADE')
+    op.create_index(op.f('ix_speakers_sessions_version_operation_type'), 'speakers_sessions_version', ['operation_type'], unique=False)
+    op.create_index(op.f('ix_speakers_sessions_version_transaction_id'), 'speakers_sessions_version', ['transaction_id'], unique=False)
+    op.create_foreign_key(None, 'tax', 'events', ['event_id'], ['id'], ondelete='CASCADE')
```
Comment (shubham-padia) [inline migrations/versions/e9d61dbd0cb8_.py]: I think the create_foreign_key changes are not related to the PR and these keys are already there in eventyay db afaik.@SaptakS Can you please check these changes :)

Comment (SaptakS) [reply migrations/versions/e9d61dbd0cb8_.py]: Updated
